README.md

```diff
@@ -346,6 +352,13 @@ hook = "cp -f %f ~/.config/sway/config && swaymsg reload"
   So the hook does a copy of the selected theme and replaces
   `~/.config/alacritty/colors.toml`.
 
+## Debugging
```

Comment:
Debugging sounds a bit misleading to me, but I also can not think of a better word

Reply:
:+1:  I'll add some extra information to try to make it less ambiguous.